.github/workflows/build-image.yml

```diff
@@ -70,8 +66,8 @@ jobs:
           GIT_DESCRIBE_TAG="$(git describe --tags --abbrev=0)"
           GIT_DESCRIBE_TAG="${GIT_DESCRIBE_TAG:1}" # remove leading 'v'
           if [[ ! $GIT_DESCRIBE_TAG =~ [a-z] ]]; then
-            rapids-logger "Most recent tag is for release, adding `rapidsai` channel and removing `rapidsai-nightly` and `dask/label/dev` channels."
-            sed -i 's|rapidsai-nightly|rapidsai|;\|dask/label/dev|d' context/condarc
+            rapids-logger "Most recent tag is for release, adding `rapidsai` channel and removing `rapidsai-nightly` channel."
```

Comment:
Was dask-sql the only package we needed this channel for? Are packages like dask, distributed either (1) not installed or (2) available from other channels?

Reply:
Oh this is a good point. The `dask/label/dev` channel does have dask/distributed alphas which are [used in builds](https://github.com/rapidsai/docker/actions/runs/9676448229/job/26696030096#step:9:626). I thought we added this channel just for `dask-sql`, but looking through git history, that doesn't seem to be the case, so I will revert this change.